tasks/9_init.yml

```diff
@@ -9,25 +9,35 @@
   when:         ansible_os_family == "RedHat"
   notify:       restart_seafile
 
-- name:         provision init script
+- name:         provision sysvinit script
   template:
     src:        init/{{ item.name }}.initd.{{ ansible_os_family }}
     dest:       /etc/init.d/{{ item.name }}
     owner:      root
     group:      root
     mode:       0750
-  when:         ansible_os_family in item.os_family
+  when:         ansible_service_mgr == "sysvinit"
```

Comment:
You here remove the logic conditional handling `SEAFILE_INIT_SCRIPTS` where Red Hat has two sysv init scripts, and Debian only one. Is that intentionally? I'd expect this should be
`ansible_os_family in item.os_family and ansible_service_mgr == "sysvinit"`